thermo_scenes/scripts/preprocess_thermal.py

```diff
@@ -37,7 +37,9 @@ def main() -> None:
     flir.save_normalised_thermal_images(
         paths.path_to_thermal_images_curated, paths.path_to_csv_files
     )
-    flir.save_temperature_bounds(paths.path_to_txt)
+    flir.save_temperature_bounds(
+        str(Path(paths.path_to_json, "temperature_bounds.json"))
+    )
```

Comment:
Couldn't we have those function private and have one public function htat saves everything in a "output_folder" and creates the right subfolders?